### ISSUE TITLE
Not Working ??

### ISSUE THREAD

**Issue by lauro86** [BarleyFlour/RespecMod#80]
Hello all i was wondering i know this is just for ISSUES but for some reason its not working .. i click on Download and then i get the file **RespecMod-master** and i install it with nexus mod , but it **never** says "**OK**"  and doesn't work... am i doing something Wrong ?

**Comment by Pheace** [comment]
I agree it's confusing. I doubt many people are getting this to work without help. 

On the Code tab go to Releases on the right and click on the latest release there and download Respecwrath.zip. That has the DLL/Json that the mod needs as well, those need to be put into the RespecWrath folder. 

Even then I'm not sure the mod works while it's still in the respecmod-master folder -> respecwrath. 

I think I just extracted the code one, then rezipped the Respecwrath folder from that one by itself and then installed with the nexus mod (so it recognizes it as installed), then added the dll/json from the respecwrath.zip to the Respecwrath folder. 

Maybe it should work fine from the master folder as well but this worked for me so I'm good.

**Comment by vikigenius** [comment]
You have to download from the releases link. Don't download master.

**Comment by lauro86** [comment]
> You have to download from the releases link. Don't download master.



> You have to download from the releases link. Don't download master.

Well now i feel like a dumb person i just  i didnt notice the "release number" was also a link ...
